### PR TITLE
[T2A2][W09-A4]Muhammad Mustaqiim Bin Muhar

### PR DESCRIPTION
**Pull request by mustaqiimuhar**

### CODE REVIEW

**Comment by mustaqiimuhar** [comment]
Ready for review.

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -188,12 +188,19 @@
      * This is a subset of the full list. Deleting persons in the pull list does not delete
      * those persons from this list.
      */
-    private static ArrayList<String[]> latestPersonListingView = getAllPersonsInAddressBook(); // initial view is of all
+    private static ArrayList<String[]> _latestPersonListingView = getAllPersonsInAddressBook(); // initial view is of all
```
Comment (hwkchia) [inline src/seedu/addressbook/AddressBook.java:191]: 👍 Good adoption of Java standard for private variables.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -188,12 +188,19 @@
...
 
     /**
+     * Added constants to indicate the number of constants passed.
+     */
+    private static final int ARGC_INVALID = 2;
+    private static final int ARGC_GIVEN_FILE = 1;
+    private static final int ARGC_DEFAULT_FILE = 0;
+    
+    /**
```
Comment (hwkchia) [inline src/seedu/addressbook/AddressBook.java:200]: You handled the magic numbers.  But why use argc?  Typically, argc means number of arguments.  You can make it even more meaningful.

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -1150,8 +1157,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replace(prefix, "");
```
Comment (hwkchia) [inline src/seedu/addressbook/AddressBook.java:1161]: You need to get the intended purpose of the method correctly.  We need to deal with the case of replacing only the prefix, and a prefix can occur only at the start of fullString.

**Pull request closed without merging**